index.html

```diff
@@ -1738,88 +1738,65 @@
-target of harassment.
-
-<div class="practice">
-  <p>
-    <span class="practicelab" id="abuse-reporting">
-      Systems that allow for communicating on the Web must provide an effective capability to report
-      abuse.
-    </span>
-  </p>
-</div>
-
 Reporting mechanisms are mitigations, but may not prevent harassment, particularly in cases where
 hosts or intermediaries are supportive of or complicit in the abuse.
 
-<div class="note">
-  Effective reporting is likely to require:
-
-  * standardized mechanisms to identify abuse reporting contacts
-  * visible, usable ways provided by sites and user agents to report abuse
-  * identifiers to refer to senders and content
-  * the ability to provide context and explanation of harms
-  * people responsible for promptly responding to reports
-  * tools for pooling mitigation information (see Unwanted information, below)
-</div>
-
-## Unwanted Information {#unwanted-information}
-
-Receiving unsolicited information that either may cause distress or waste the recipient's
-time or resources is a violation of privacy.
+Effective reporting is likely to require:
 
-<div class="practice">
+* standardized mechanisms to identify abuse reporting contacts;
+* visible, usable ways provided by sites and user agents to report abuse;
+* identifiers to refer to senders and content;
+* the ability to provide context and explanation of harms;
+* people responsible for promptly responding to reports;
+* tools for pooling mitigation information (see [[[#example-10]]]).
 
-<p>
-  <span class="practicelab" id="principle-protect-unwanted-information">
-    [=User agents=] and other [=actors=] should take
-    steps to ensure that their [=user=] is not exposed to unwanted information. Technical standards
-    must consider the delivery of unwanted information as part of their architecture and must
-    mitigate it accordingly.
-  </span>
-</p>
-</div>
+<aside class="note">
+  Some useful research overviews of online harassment include: [[?PEW-Harassment]],
+  [[?Addressing-Cyber-Harassment]] and [[?Internet-of-Garbage]].
+</aside>
 
 <dfn>Unwanted information</dfn> covers a broad range of unsolicited communication, from messages
 that are typically harmless individually but that become a nuisance in aggregate (spam) to the
-sending of images that will cause shock or disgust due to their graphic, violent, or explicit nature
-(e.g. pictures of one's genitals). While it is impossible, in a communication system involving many
-[=people=], to offer perfect protection against all kinds of unwanted information, steps can be
-taken to make the sending of such messages more difficult or more costly, and to make the senders
-more accountable. Examples of mitigations include:
+sending of explicit, graphic, or violent images.
 
-* Restricting what new users of a service can post, notably limiting links and media until they have
+<aside class="example">
```

Comment:
To make the reference more stable:

```suggestion
<aside class="example" id="example-reducing-unwanted-information">
```